src/Configuration/Config/ref/Microsoft.Extensions.Configuration.netstandard2.0.cs

```diff
@@ -6,10 +6,12 @@ namespace Microsoft.Extensions.Configuration
     public static partial class ChainedBuilderExtensions
     {
         public static Microsoft.Extensions.Configuration.IConfigurationBuilder AddConfiguration(this Microsoft.Extensions.Configuration.IConfigurationBuilder configurationBuilder, Microsoft.Extensions.Configuration.IConfiguration config) { throw null; }
+        public static Microsoft.Extensions.Configuration.IConfigurationBuilder AddConfiguration(this Microsoft.Extensions.Configuration.IConfigurationBuilder configurationBuilder, Microsoft.Extensions.Configuration.IConfiguration config, bool shouldDisposeConfiguration) { throw null; }
```

Comment:
@HaoK thoughts?

Reply:
Seems fine, pattern has been to just add overloads like this that set source properties